self-paced-labs/vertex-ai/vertex-ai-qwikstart/lab_exercise.ipynb

```diff
@@ -2,60 +2,72 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.                        validation_data=evalds,
</pre><p>evalds here should be devds</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1570/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>